src/components/Interviewer/Pool/IntervieweeAvailability.tsx

```diff
@@ -0,0 +1,129 @@
+import UIMainContainer from 'components/UI/UIBoxContainer';
+import React, { useEffect, useState } from 'react';
+import { IntervieweePoolAvailabilityProps } from 'utils/ts/propsInterfaces';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import {GrNext, GrPrevious} from 'react-icons/gr'; 
+import { Button } from '@material-ui/core';
+import DateTime from 'utils/helpers/DateTime';
+import { off } from 'process';
+
+const IntervieweeAvailability: React.FC<IntervieweePoolAvailabilityProps> = ({
+  poolCandidate,
+  setSelectedDateOfInterview
+}) => {
+  const [currentStartWeek, setCurrentStartWeek] = useState(getStartWeekDay(new Date().toLocaleString("en-US", {timeZone: "America/Los_Angeles"})));
+
+  function getStartWeekDay(date: string) {
```

Comment:
Maybe this function could be reused in the future, do we need to move this to utils? cc/ @GoberInfinity